.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
@andrii-antiukhov Is it necessary to check these files in?

Reply:
Sorry, i do not know. It was changed not by me, maybe it should not be in GIT, but for this we should exclude this files .idea from GIT in .gitignore , maybe.